backend/src/org/commcare/xml/SuiteParser.java

```diff
@@ -118,9 +130,10 @@ public Suite parse() throws InvalidStructureException, IOException,
                         Menu m = new MenuParser(parser).parse();
                         menus.addElement(m);
                     } else if (parser.getName().toLowerCase().equals("fixture")) {
-                        //this one automatically commits the fixture to the global memory
-                        if (!inValidationMode()) {
-                            new FixtureXmlParser(parser, false, fixtureStorage).parse();
+                        if (!isValidationPass) {
+                            // commit fixture to the memory, overwriting existing
+                            // fixture only during install, not init or validation
+                            new FixtureXmlParser(parser, !skipResources, fixtureStorage).parse();
```

Comment:
The fix is to pass `!skipResources` in as the `overwrite` argument to the fixture parser. `!skipResources` is `true` only when suite file is installed. It is `false` on suite initialization and validation passes.